contracts/cw721-poap/rustfmt.toml

```diff
@@ -0,0 +1,15 @@
+# stable
```

Comment:
Can we move this file to the project root directory so that we have a common style for each contract?

Reply:
Removed all rustfmt in the contracts.